version.go

```diff
@@ -0,0 +1,3 @@
+package ccloud
+
+const SDKVersion = "v0.1.0"
```

Comment:
We need to make sure our CI bumps this version automatically. This should be updated manually in the meantime.